demo-java/app/src/gms/java/com/example/placesdemo/AutocompleteAddressActivity.java

```diff
@@ -131,15 +151,27 @@ public void onMapReady(GoogleMap googleMap) {
                     MapStyleOptions.loadRawResourceStyle(this, R.raw.style_json));
 
             if (!success) {
-                Log.e(TAG, "Style parsing failed.");
+                e(TAG, "Style parsing failed.");
             }
         } catch (Resources.NotFoundException e) {
-            Log.e(TAG, "Can't find style. Error: ", e);
+            e(TAG, "Can't find style. Error: ", e);
         }
         map.moveCamera(CameraUpdateFactory.newLatLngZoom(coordinates, 15f));
         marker = map.addMarker(new MarkerOptions().position(coordinates));
     }
 
+    public void onCheckboxClicked(View view) {
+        // Is the view now checked?
+        boolean checked = ((CheckBox) view).isChecked();
+
+        // Check which checkbox was clicked
+        switch (view.getId()) {
```

Comment:
I think only one `Checkbox` has this as an onClick listener? If so no need to `switch` on the view's ID.

Reply:
obsolete when using `setOnCheckedChangeListener`